.changeset/bright-seas-march.md

```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/confirmation-modal': minor
+---
+
+- Dropped the `isRequired` from the `buttonText` prop type. As of [version 5.1.0](https://github.com/mongodb/leafygreen-ui/blob/main/packages/confirmation-modal/CHANGELOG.md#510), `buttonText` was marked as a deprecated prop that is now optional.
```

Comment:
I think that these should all be present tense verbs, so `Drops the ...`, `Fixes existing ...`, and `Adds missing ...`